python/test/test_meas.py

```diff
@@ -7,21 +7,30 @@ def test_gaussian_curve():
     meas = dp.c.make_zCDP_to_approxDP(dp.m.make_gaussian(*input_space, 4.))
     profile = meas.map(d_in=1.)
     assert profile.epsilon(delta=0.) == float('inf')
-    assert profile.epsilon(delta=1e-3) == 0.6880024554878086
+    assert profile.epsilon(delta=1e-3) == 0.6880024554878085
```

Comment:
Was the old value actually wrong? Should the test be less strict?

Reply:
They are both valid upper bounds. This calculation seems to be very slightly more numerically stable.